apps/stress-test/pageConfig.js

```diff
@@ -0,0 +1,86 @@
+const glob = require('glob');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+/**
+ * Automatically configures Webpack to find pages so
+ * developers can just add new pages without touching
+ * the Webpack config.
+ */
+
+/**
+ * Find all the source pages and map them to
+ * config objects used to generate the Webpack config.
+ */
+const getPages = () => {
```

Comment:
would be nice to add some types with jsdoc here, we've been doing that with our build scripts for the last few months and it makes maintenance a lot easier

Reply:
done!